ci/check_dump_cores_linux.sh

```diff
@@ -3,31 +3,24 @@
 
 path="$1"
 binary="$2"
-logfile="$3"
+processId="$3"
+logfile="$4"
 cd $path
-ls -lrt core
+ls -lrt /tmp/pxscenecrash
 retVal=$?
 if [ "$retVal" -eq 0 ]
 	then
-	tempLogFile="gdb.txt"
-	coreFileName="core"
-	sudo chmod 777 $coreFileName
-	echo "gdb -q --command=\"$TRAVIS_BUILD_DIR/ci/debuggercmds_linux\" -c $coreFileName $binary  2&>gdblogs"
-	gdb -q --command="$TRAVIS_BUILD_DIR/ci/debuggercmds_linux" -c $coreFileName $binary  2&>gdblogs
-	cat gdblogs
-	if [ "$TRAVIS_PULL_REQUEST" != "false" ]
+        sudo gdb -q --command="$TRAVIS_BUILD_DIR/ci/debuggercmds_linux" --pid=$3 $2 2&> gdblogs
+        if [ "$TRAVIS_PULL_REQUEST" != "false" ]
 		then
 		echo "**********************PRINTING CORE STACK DETAILS**************************"
-		echo "PATH :  $path"
-		cat $tempLogFile
+		cat gdblogs
```

Comment:
please clarify the difference between gdb.txt output and gdblogs output. is it ok, if we don't have gdb.txt?

Reply:
This is actually a code cleanup. gdb.txt is stored in a variable tempLogFile, but there is no data redirected to tempLogFile or gdb.txt. The output of gdb is sent to gdblogs. So i removed the unused variables.